mlir/test/Dialect/Vector/vector-transfer-flatten.mlir

```diff
@@ -188,9 +188,31 @@ func.func @transfer_read_leading_dynamic_dims(
 
 // -----
 
-// One of the dims to be flattened is dynamic - not supported ATM.
+// The vector could be a non-contiguous slice of the input
+// memref.
```

Comment:
Could you explain _why_ "could"?

Reply:
Are you objecting about the word "could"? Perhaps you prefer "is" ?